app/scripts/controllers/SysinfoController.coffee

```diff
@@ -103,5 +105,16 @@ angular.module('neo4jApp.controllers')
       $scope.autoRefresh = !$scope.autoRefresh
       refreshLater()
 
+    $scope.isCurrentConnection = (addresses) ->
+      if Settings.useBolt
+        boltHost = Settings.boltHost || $rootScope.boltHost
+        boltAddressForMember = Utils.getServerAddressByProtocol('bolt', addresses)[0]
+        return boltAddressForMember && boltAddressForMember.includes(boltHost)
+      else
+        Utils.getServerAddressByProtocol($location.protocol(), addresses).includes($location.host)
```

Comment:
Same reuse here but for REST https://github.com/neo4j/neo4j-browser/blob/2.0/app/scripts/services/UtilityREST.coffee#L219